src/docs/markdown/caddyfile/options.md

```diff
@@ -83,27 +123,84 @@ Specifies an External Account Binding to use for all ACME transactions.
-
 ##### `cert_issuer`
 Defines the issuer (or source) of TLS certificates.
+
+
+
+## Server Options
+
+Allows configuring options of the [HTTP servers](/docs/json/apps/http/servers/) that can't otherwise be provided in site blocks.
+
+Can be specified more than once, with different `listener_address` values, to configure different options per server. For example, `servers :443` will only apply to the server that is bound to the listener address `:443`. Omitting the listener address will apply the options to any remaining server.
+
+<aside class="tip">
+	Use the <a href="/docs/command-line#caddy-adapt"><code>caddy adapt</code></a> command to find the listen address for the servers in your Caddyfile.
+</aside>
+
+For example, to configure different options for the servers on port `:80` and `:443`, you would specify two `servers` blocks:
+
+```caddy
+{
+	servers :443 {
+		protocol {
+			experimental_http3
+		}
+	}
+
+	servers :80 {
+		protocol {
+			allow_h2c
+		}
+	}
+}
+```
+
+##### `listener_wrappers`
+
+Allows configuring [listener wrappers](/docs/json/apps/http/servers/listener_wrappers/), which can modify the behaviour of the base listener. They are applied in the given order.
+
+There is a special no-op [`tls`](/docs/json/apps/http/servers/listener_wrappers/tls/) listener wrapper provided as a standard module which marks where TLS should be handled in the chain of listener wrappers. It should only be used if another listener wrapper must be placed in front of the TLS handshake.
+
+For example, assuming you have the [`proxy_protocol`](/docs/json/apps/http/servers/listener_wrappers/proxy_protocol/) plugin installed:
+
+```caddy-d
+listener_wrappers {
+	proxy_protocol {
+		timeout 2s
+		allow 192.168.86.1/24 192.168.86.1/24
+	}
+	tls
+}
+```
+
+##### `timeouts`
+
+- **read_timeout** is a [duration value](/docs/conventions#durations) that sets how long to allow a read from a client's upload. Setting this to a short, non-zero value can mitigate slowloris attacks, but may also affect legitimately slow clients. Defaults to no timeout.
+
+- **read_header_timeout** is a [duration value](/docs/conventions#durations) that sets how long to allow a read from a client's request headers. Defaults to no timeout.
+
+- **write_timeout** is a [duration value](/docs/conventions#durations) that sets how long to allow a write to a client. Note that setting this to a small value when serving large files may negatively affect legitimately slow clients. Defaults to no timeout.
+
+- **idle_timeout** is a [duration value](/docs/conventions#durations) that sets the maximum time to wait for the next request when keep-alives are enabled. Defaults to 5 minutes to help avoid resource exhaustion.
+
+##### `max_header_size`
+The maximum size to parse from a client's HTTP request headers. It accepts all formats supported by [go-humanize](https://github.com/dustin/go-humanize/blob/master/bytes.go).
```

Comment:
```suggestion
##### `max_header_size`

The maximum size to parse from a client's HTTP request headers. It accepts all formats supported by [go-humanize](https://github.com/dustin/go-humanize/blob/master/bytes.go).
```

Reply:
I was matching the spacing on the ones in other sections, but yeah that's fine

Reply:
Hmm are you sure? I thought that's what I was doing.